src/ui/SmartSnippet/SmartSnippetCollapsibleSuggestion.ts

```diff
@@ -186,4 +196,18 @@ export class SmartSnippetCollapsibleSuggestion {
       this.checkbox.el
     );
   }
+
+  private sendOpenSourceAnalytics(element: HTMLElement, href: string) {
+    return this.bindings.usageAnalytics.logCustomEvent<IAnalyticsSmartSnippetSuggestionOpenSourceMeta>(
```

Comment:
You should be able to do just: 
```suggestion
    return this.usageAnalytics.logCustomEvent<IAnalyticsSmartSnippetSuggestionOpenSourceMeta>(
```

Reply:
CollapsibleSuggestion doesn't extend `Component`.